app/src/main/kotlin/com/wire/android/ui/e2eiEnrollment/GetE2EICertificateUI.kt

```diff
@@ -43,7 +43,7 @@ fun GetE2EICertificateUI(
     LaunchedEffect(Unit) {
         viewModel.requestOAuthFlow.onEach {
             OAuthUseCase(context, it.target, it.oAuthClaims, it.oAuthState).launch(
-                context.getActivity()!!.activityResultRegistry
+                context.getActivity()!!.activityResultRegistry, forceLoginFlow = true
```

Comment:
`OAuthUseCase.launch` so the parameter `forceLoginFlow` is always true. What is the point of it then?